iothub/service/src/Device.cs

```diff
@@ -110,5 +110,13 @@ public Device(string id)
         /// </summary>
         [JsonProperty(PropertyName = "deviceScope", NullValueHandling = NullValueHandling.Ignore)]
         public virtual string Scope { get; set; }
+
+        /// <summary>
+        /// The DTDL model id provided by the device during the device connection.
+        /// The value will be null for a non-pnp device.
+        /// The value will be null for a pnp device until the device connects and registers with the model id.
```

Comment:
This comment was stated for twins but i'm not sure if it applies to device identity as well. I will update it if someone knows better.

Reply:
Sounds good.

Reply:
This change needs to be a part of `Module` class as well.

Reply:
I would suggest including all three in this PR:
- model Id for device and module types
- model Id for twin type
- service API bump

Reply:
I was planning to do it in separate PR as you mentioned this morning Twin is in the Common nuget. Let me evaluate if making incremental changes will help or not and proceed accordingly.